deepoctl/workflow_abstraction.py

```
@@ -105,8 +105,11 @@ def __init__(self, recognition_version_id, amqp_url, routing_key):
                 recognition_version_id = int(recognition_version_id)
                 self._recognition = self._client.create_recognition(version_id=recognition_version_id)
             except ValueError:
-                logging.warning("Cannot cast recognition ID into a number")
+                logging.error("Cannot cast recognition ID into a number")
```

Is it something that can happen ? Don't we want to make let it raise ?